packages/client/test/sim/README.md

```diff
@@ -0,0 +1,81 @@
+### Shandong sim setup
+
+This sim test is to setup a single ethereumjs<>lodestar instance to allow executing testvectors for the EIPs targetting Shanghai hardfork.
+
+### Prerequisite(s)
+
+- Docker since lodestar docker image is used to run CL to drive the post-merge sim run
+- Docker should run without `sudo`, else prefix `sudo` in from of docker commands in `test/sim/single-run.sh`
+
+### How to run
+
+1. `npm install` the `ethereumjs-monorepo`
+2. `cd packages/client`
+3. `docker pull chainsafe/lodestar:latest`
+4. Install linux package `jq` is not installed
+5. Create a data directory `data/shandong` (or any other place convinient for you).
+
+#### Auto run using script
+
+(Might only run good on ubuntu/linux)
+
+Currently you can just start the local instance by
+`DATADIR=data/shandong test/sim/./single-run.sh`
+
+This command run should start both ethereumjs and lodestar in terminal. Soon you should see lodestar driving ethereumjs in PoS configuration.
+
+##### Process cleanup
+
+The script should auto clean the processes. In case it fails to do so:
+
+1. Remove lodestar by `docker rm -f beacon`
+2. Find the ethereumjs process by doing `ps -a | grep client` and do `kill <process id>`
+
+#### Manual run using script
+
+1. Clean DATADIR (`data/shandong`) in start of each run and do `mkdir data/shandong/ethereumjs && mkdir data/shandong/lodestar`
+2. Start ethereumjs: `npm run client:start -- --datadir data/shandong/ethereumjs --gethGenesis test/sim/configs/geth-genesis.json --rpc --rpcEngine --rpcEngineAuth false`
+3. Get genesis hash from `ethereumjs` client:
+
+```
+curl --location --request POST 'http://localhost:8545' --header 'Content-Type: application/json' --data-raw '{
+      "jsonrpc": "2.0",
+      "method": "eth_getBlockByNumber",
+      "params": [
+          "0x0",
+          true
+      ],
+      id: 1
+    }' 2>/dev/null | jq ".result.hash"
+```
```

Comment:
Sorry, I know this is just convenience-posting, just put my result on this (for sure can do in another way though):

```shell
curl --location --request POST 'http://localhost:8545' --header 'Content-Type: application/json' --data-raw '{"jsonrpc": "2.0", "method": "eth_getBlockByNumber","params": ["0x0",true],id: 1}' 2>/dev/null | jq ".result.hash"
```

```shell
parse error: Invalid numeric literal at line 1, column 10
```

Reply:
Ah, got it, the `id` in the above command needs to get double paranthesis, so `"id"`.

Reply:
oops add id in quotes: "id"